docs/sharing/publish-storybook.md

```diff
@@ -129,9 +129,39 @@ When you publish Storybook, you also get component history and versioning down t
 
 ![Library history in Chromatic](./workflow-history-versioning.png)
 
+## Publish to GitHub Pages
+
+To deploy to GitHub Pages, simply use the [deploy-storybook-to-github-pages](https://github.com/marketplace/actions/deploy-storybook-to-github-pages) GitHub Action.  
+
+From your project's root directory, add a new file called `deploy-ghp.yml` inside the `.github/workflows` directory:
+<!-- prettier-ignore-start -->
+
+<CodeSnippets
+  paths={[
+    'common/ghp-github-action.js.mdx',
+  ]}
+/>
+
+<!-- prettier-ignore-end -->
+
+
+It does the build for you, and it publishes the built site to GitHub pages with the Actions based pages deployment. 
+
+The action has many customizable options including `install_command` (default is `npm ci`) and `build_command` (default is `npm run build-storybook`) and output directory.
+
+
+<div class="aside">
+
+ℹ️ Be sure to [set up your project](https://github.com/marketplace/actions/deploy-storybook-to-github-pages#set-up-your-project-for-actions-deployed-pages) for Actions-deployed pages!
+
+</div>
+
+Commit and push the file. Congratulations, you've successfully automated publishing your Storybook to GitHub Pages!
+
+
 ## Publish Storybook to other services
 
-Since Storybook is built as a static web application, you can also publish it to any web host, including [GitHub Pages](https://docs.github.com/en/pages), [Netlify](https://www.netlify.com/), [AWS S3](https://aws.amazon.com/s3/), and more. However, features such as [Composition](./storybook-composition.md),
+Since Storybook is built as a static web application, you can also publish it to any web host, including [Netlify](https://www.netlify.com/), [AWS S3](https://aws.amazon.com/s3/), and more. However, features such as [Composition](./storybook-composition.md),
 [embedding stories](./embed.md), history, and versioning require tighter integration with Storybook APIs and secure authentication. Your hosting provider may not be capable of supporting these features. Learn about the Component Publishing Protocol (CPP) to see what.
```

Comment:
@mickmcgrath13 we have a bit of room for improvement here, I was wondering if you'd be okay with adjusting this section to the following:

```md
## Publish Storybook to other services

Since Storybook is built as a static web application, you can also publish it to any web host, including [GitHub Pages](https://docs.github.com/en/pages), [Netlify](https://www.netlify.com/), [AWS S3](https://aws.amazon.com/s3/), and more. However, features such as [Composition](./storybook-composition.md),
[embedding stories](./embed.md), history, versioning, and assets may require tighter integration with Storybook APIs and secure authentication. If you want to know more about headers, you can refer to the [Migration guide](https://github.com/storybookjs/storybook/blob/main/MIGRATION.md#deploying-build-artifacts). Additionally, if you want to learn about the Component Publishing Protocol (CPP), you can find more information below.

### GitHub Pages

To deploy Storybook on GitHub Pages, use the community-built [Deploy Storybook to GitHub Pages](https://github.com/bitovi/github-actions-storybook-to-github-pages) Action. To enable it, create a new workflow file inside your `.github/workflows` directory with the following content:


<!-- prettier-ignore-start -->

<CodeSnippets
  paths={[
    'common/ghp-github-action.js.mdx',
  ]}
/>

<!-- prettier-ignore-end -->

<div class="aside">

ℹ️ The GitHub Pages Action requires additional configuration options to customize the deployment process. Refer to the [official documentation](https://github.com/marketplace/actions/deploy-storybook-to-github-pages) for more information.

</div>

Reply:
updated as suggested